effect_parameter_knob.xml

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE template>
```

Comment:
this file is in the wrong place.
please apply the changes to res/skins/Deere/effect_parameter_knob.xml which you initially modified.